sdcm/sct_runner.py

```diff
@@ -351,160 +460,138 @@ def _get_base_image(self, image=None):
 
 
 class GceSctRunner(SctRunner):
-    """Provisions and configures the SCT runner on GCE"""
+    """Provision and configure the SCT runner on GCE."""
     BASE_IMAGE = "https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts"
     SOURCE_IMAGE_REGION = "us-east1"  # where the source Runner image will be created and copied to other regions
+    IMAGE_BUILDER_INSTANCE_TYPE = "e2-standard-2"
+    REGULAR_TEST_INSTANCE_TYPE = "e2-standard-4"  # 2 vcpus, 16G
+    LONGTERM_TEST_INSTANCE_TYPE = "e2-standard-2"  # 2 vcpus, 8G
```

Comment:
Instance type for longterm worse than for regular case. Is it intentional?

Reply:
Nope. Thank you for your keen eyes!